### DISCUSSION TITLE
Holding a dash button causes catcher to blink stronger

### DISCUSSION THREAD

**Discussion by Kimitakari** [ppy/osu#16432] [Q&A]
I find this a bit hurtful when you stare at your catcher for a plenty of time. Although with default skin the difference in both osu! clients hardly visible, the difference of using a custom skin is really noticeable and causes a constant blicking in lazer.

Lazer:
![blink custom skin lazer](https://user-images.githubusercontent.com/36848514/149137251-301a6b65-e8ed-4522-8804-92f15f9d4b5a.gif)
![blink default skin lazer](https://user-images.githubusercontent.com/36848514/149137256-a809e107-9385-417e-bdac-a278897ba3c1.gif)

Stable:
![blink custom skin stable](https://user-images.githubusercontent.com/36848514/149137266-c48a3dc0-559c-4937-a124-4b3a63ac0e85.gif)
![blink default skin stable](https://user-images.githubusercontent.com/36848514/149137269-6ed9f6ec-906a-4aa4-adc7-4927a6e14cc6.gif)

**Comment by bdach** [comment]
can you attach the skin for testing purposes?

**Reply by Kimitakari** [reply]
Here https://drive.google.com/file/d/1cvvPRtsZ9cGaDvKQnU1QW4ZafmY2XiGD/view

**Reply by peppy** [reply]
I think this is fine as is.
